io.github.davidoc26.wallpaper_selector.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app-id": "io.github.davidoc26.wallpaper_selector",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "42",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "wallpaper-selector",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--talk-name=org.freedesktop.Flatpak"
```

Comment:
why do you need this DBus access?

Reply:
As I wrote, this is necessary in order to set the desktop wallpaper via cli, for example using gsettings in gnome

Reply:
There's a bunch of wallpaper apps like https://beta.flathub.org/apps/details/im.bernard.Nostalgia which seem to use libportal for setting wallpaper.